docs/tremor-script/index.md

```diff
@@ -116,9 +116,12 @@ The language does not support literal character or Unicode code-points at this t
 For strings tremor allows string interpolation, this means embedding code directly into strings to create strings out of them. This is a convenience as an alternative of using the string::format function.
```

Comment:
i think we need to either remove the mention of `string::format` or clarify the differences that we have now.

Reply:
I think the sentence holds true it calls it out as an alternative but it also doesn't add much value so lets yank it and make sure no one is thrown of by it :+1: